cpp/modules/deck.gl/core/src/lib/attribute/attribute-manager.h

```diff
@@ -67,7 +67,7 @@ class AttributeManager {
   void invalidate(const std::string& attributeName);
   void invalidateAll();
 
-  auto update(const std::shared_ptr<arrow::Table>& table) -> std::shared_ptr<arrow::Table>;
+  auto update(const std::shared_ptr<arrow::Table>& table) -> std::shared_ptr<lumagl::WebGPUTable>;
```

Comment:
My sense is that this class should have the GPUTable as a (public) member, and `update()` should return void.